src/twisted/internet/base.py

```diff
@@ -1168,6 +1168,10 @@ def getDestination(self):
             reflect.qual(self.__class__) + " did not implement "
             "getDestination")
 
```

Comment:
You will need an additional blank like here.

Reply:
Hi, thanks for the review. I fixed all the issues except this one as I am not clear on where to add the blank line. It currently seems consistent with the rest of the code to me.

Reply:
Hmm, this is odd. Normally spacing in Twisted is 3-2-1 (3 blank lines at module level, 2 at class level, 1 within functions and methods). I think that the linter is checking just the diff, so it isn't complaining here. If it doesn't care I don't, particularly in light of #1049. I just need a build which passes in order to merge.

Reply:
So what can I do about it? I don't understand what the issue is from the CI output.